notebooks/ivf_flat_example.ipynb

```diff
@@ -0,0 +1,803 @@
+{
```

Comment:
<p>Can you add a little more detail here about what the notebook accomplishes? Doesn't need to be extensive, but at least somehting like "It builds and search an index using a dataset from the <a href="https://github.com/erikbern/ann-benchmarks/#data-sets" target="_blank">ann-benchmarks million-scale datasets</a>, saves/loads the index to disk, and explores some of the important parameters for fine-tuning the performance and accuracy of the index."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>added</p>